packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+import os
+import json
+import subprocess
+import requests
+from pathlib import Path
+from datetime import datetime
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)